src/Microsoft.AspNet.Razor/Generator/Compiler/CodeTree/Chunks/TagHelpers/TagHelperChunk.cs

```diff
@@ -12,6 +12,29 @@ namespace Microsoft.AspNet.Razor.Generator.Compiler
     public class TagHelperChunk : ChunkBlock
     {
         /// <summary>
+        /// Instantiates a new <see cref="TagHelperChunk"/>.
+        /// </summary>
+        /// <param name="tagName">The tag name associated with the tag helpers HTML element.</param>
+        /// <param name="attributes">The attributes associated with the tag helpers HTML element.</param>
+        /// <param name="selfClosing">
+        /// The <c>bool</c> indicating whether or not the tag of the tag helpers HTML element is self-closing.
```

Comment:
- use the same doc for this parameter (probably most parameters) as in `TagHelperExecutionContext`